github-archive/webapp/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python2.7 -m SimpleHTTPServer 8080
```

Comment:
if you want to make it nicer, add a tornado app with a staticfilehandler :dancer: